mkversion.sh

```diff
@@ -131,7 +133,10 @@ cat <<EOF > "$PKG_BUILDDIR/cmd/VERSION"
 $v
 EOF
 
+fmts=$(go run -mod=vendor ${RELATIVE}/asserts/info)
```

Comment:
This will fail on systems where we don't have vendored dependencies. Maybe something like this:
```suggestion
GORUNFLAGS=${GORUNFLAGS--mod=vendor}
fmts=$(go run $GORUNFLAGS ${RELATIVE}/asserts/info)
```